src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -106,7 +109,18 @@ function ReportActionItemMessageEdit(
 
     useEffect(() => {
         const parser = new ExpensiMark();
-        const originalMessage = parser.htmlToMarkdown(action.message?.[0]?.html ?? '');
+        const videoAttributesCache: Record<string, string> = {};
+        const originalMessage = parser.htmlToMarkdown(action.message?.[0]?.html ?? '', {
+            cacheVideoAttributes: (videoSource, attrs) => {
+                videoAttributesCache[videoSource] = attrs;
```

Comment:
@dominictb Hi, Why don't we directly set `attrs` to `draftMessageVideoAttributeCache`? With this, we can move the `clear` function to above